web-ui/src/main/resources/catalog/style/gn.less

```diff
@@ -1,18 +1,10 @@
 // GN Styles
 @import "../lib/style/font-awesome/less/font-awesome.less";
-@import "gn_bootstrap.less";
-
+@import "../lib/style/bootstrap/less/variables.less";
 @import (less) "../lib/ol.css";
 @import (less) "../lib/angular.ext/hotkeys/hotkeys.min.css";
-@import "../lib/bootstrap.ext/tagsinput/bootstrap-tagsinput.less";
-// import the css files as less files so they are included in the final output of wro4j
-@import "../lib/bootstrap.ext/datepicker/datepicker.less";
-@import (less) "../lib/style/datetimepicker.css";
 @import "gn_doc.less";
-@import "gn_icons.less";
-
 @import "ellipsis.less";
-@import "timeline.less";
```

Comment:
This is removed but not added anywhere else so layout is broken in record history, harvester history

Reply:
![image](https://user-images.githubusercontent.com/1701393/67458501-a5b0b900-f636-11e9-855f-0b643e23d0a2.png)

Reply:
Should we add it back in gn-picker.less ?